shared/chat/conversation/container.tsx

```diff
@@ -22,7 +22,7 @@ type SwitchProps = {
   type: 'error' | 'noConvo' | 'rekey' | 'youAreReset' | 'normal' | 'rekey'
 }
 
-class Conversation extends React.PureComponent<SwitchProps> {
+export class Conversation extends React.PureComponent<SwitchProps> {
```

Comment:
if we don't export it, it has to copy a bunch of stuff to other files

Reply:
These individual things aren't so bad so if it comes back it's not the end of the world